src/getHumanAge.test.js

```diff
@@ -3,8 +3,39 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test('should return [0, 0] for catAge = 0 and dogAge = 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
```

Comment:
Please add test for min - 1 (15 - 1 = 14 // 0 years) to check boundaries

Reply:
Not fixed, (getHumanAge(14, 14) should return [0, 0] you don't need to verify minus values)